src/analytics.js

```diff
@@ -653,6 +666,7 @@ window.addEventListener( 'beforeunload', () => {
 window.Altis.Analytics.updateEndpoint = Analytics.updateEndpoint;
 window.Altis.Analytics.getEndpoint = Analytics.getEndpoint;
 window.Altis.Analytics.getAudiences = Analytics.getAudiences;
+window.Altis.Analytics.overrideAudiences = Analytics.overrideAudiences;
```

Comment:
Why would we always want this to be available in user land? I was thinking it's better to replace `window.Altis.Analytics.getAudiences`. The main reason is that it's an admin/editorial use-case rather than something needed for a regular visitor. I know it's not a lot of code but I'd like to keep the main `analytics.js` file size as small as possible.

Still, if it's a pain to get around then no worries. We can address the overall state of this file in a future iteration.

Reply:
For reference you could do something like:

```js
var getRealAudiences = Altis.Analytics.getAudiences;
Altis.Analytics.getAudiences = function () {
     // Return overrides if set
     if ( overrides ) {
         return overrides;
     }
     return getRealAudiences();
};
```

Reply:
Good Q; I think it is useful, and it was basically "free", but can change if you'd prefer not to expose as a public API.

Reply:
I'd prefer not to but lets leave it for now and just not document it. Want to get this over the line really.